README.md

```diff
@@ -127,22 +127,25 @@ In addition, it can also be configured by passing in a `*whisk.Config` object as
 
 ```go
 config := &whisk.Config{
-  Host: "openwhisk.ng.bluemix.net",
-  Version: "v1"
+  Host: "us-east.functions.cloud.ibm.com",
```

Comment:
I know I use "us-south" (dallas)... as it has to match my IBM Cloud account... it would be good to make note of that somehow.

Reply:
update `APIHOST` parameter example should cover the different host values.